staging/src/k8s.io/apiserver/plugin/pkg/audit/log/backend.go

```diff
@@ -82,8 +82,8 @@ func (b *backend) logEvent(ev *auditinternal.Event) {
 		ip = ev.SourceIPs[0]
 	}
 
-	line := fmt.Sprintf("%s AUDIT: id=%q ip=%q method=%q user=%q groups=%q as=%q asgroups=%q namespace=%q uri=%q response=\"%s\"\n",
-		ev.Timestamp.Format(time.RFC3339Nano), ev.AuditID, ip, ev.Verb, username, groups, asuser, asgroups, namespace, ev.RequestURI, response)
+	line := fmt.Sprintf("%s AUDIT: id=%q stage=%q ip=%q method=%q user=%q groups=%q as=%q asgroups=%q namespace=%q uri=%q response=\"%s\"\n",
```

Comment:
I think we should think a little about what the goal of this log format is before adding stage to it. Originally, I think it was trying to be consistent with basic auditing, but now that code was forked. Now, it's still missing a bunch of information from the audit event. Just thinking "out loud"...